examples/experimental/v1alpha2/backendtlspolicy-ca-certs.yaml

```
@@ -0,0 +1,17 @@
+#$ Used in:
+#$ - site-src/guides/tls.md
+apiVersion: gateway.networking.k8s.io/v1alpha2
+kind: BackendTLSPolicy
+metadata:
+  name: tls-upstream-auth
+spec:
+  targetRef:
+    kind: Service
+    name: auth-service
+    group: ""
+  tls:
+    caCertRefs:
+      - kind: ConfigMapReference
+        name: auth-cert
+        group: ""
+    hostname: auth-pod.example.com
```

Would avoid using "pod" here as it could lead to confusion
```suggestion
    hostname: auth.example.com
```